src/test/common_url_tests.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2024 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
nit: For new source files we can use a copyright header with HTTPS link and future-proof dating:
```suggestion
// Copyright (c) 2024-present The Bitcoin Core developers
// Distributed under the MIT software license, see the accompanying
// file COPYING or https://opensource.org/license/mit/.
```